roles/template_service_broker/tasks/install.yml

```diff
@@ -29,9 +29,19 @@
     - "{{ __tsb_rbac_file }}"
     - "{{ __tsb_broker_file }}"
 
+- yedit:
+    content: "{{ lookup('file', __tsb_files_location ~ '/' ~ __tsb_config_file) }}"
```

Comment:
do we still need to put this file in this repo, or something is grabbing it from origin?

Reply:
the file is currently in the repo, we will need to have a job created that will sync it from origin each time though